README.md

```diff
@@ -19,97 +20,161 @@ This WordPress project is intended to provide:
 - Homebrew (1.2.3)
   - MariaDB (10.0 or greater)
   - Wordpress Command Line (1.2.1)
+  - Composer (1.4.2)
 - PHP 7
+- Codeception for Wordpress (Wp-Browser 1.21)
+
+***
 
 ## Installation
 
-  Installation process for the setting up Wordpress on a local environment.
+Installation process for the setting up Wordpress on a local environment.
+
+- **Create the MySQL user and databases using the MariaDB monitor**.
+
+```
+mysql -uroot
+CREATE DATABASE chappress_dev;
+GRANT ALL PRIVILEGES ON chappress_dev.* TO "chappress"@"localhost" IDENTIFIED BY "chappress";
+
+CREATE DATABASE chappress_test;
+GRANT ALL PRIVILEGES ON chappress_test.* TO "chappress"@"localhost" IDENTIFIED BY "chappress";
+
+CREATE DATABASE chappress_test_wpunit;
+GRANT ALL PRIVILEGES ON chappress_test_wpunit.* TO "chappress"@"localhost" IDENTIFIED BY "chappress";
```

Comment:
Just ran tests and discovered this when they failed.

This shouldn't be necessary. `chappress_test` should be sufficient. Something is not right here.